src/modules/error/containers/__tests__/FailureContainer.spec.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import { render, fireEvent } from '@testing-library/react-native'
+import * as React from 'react'
+import TestRenderer from 'react-test-renderer'
+import typeof FailureContainer from '../FailureContainer'
+
+jest.mock('react-i18next')
+jest.mock('../../../theme/hocs/withTheme')
+
+describe('FailureContainer', () => {
+  beforeEach(() => {
+    jest.resetModules()
```

Comment:
What is the resetModules for?

Reply:
It resets the mocked modules. If you now call jest.unmock() and import a module later it won't be mocked.